frontend/src/pages/workspace/whiteboard-canvas/useCanvas.ts

```diff
@@ -58,7 +58,7 @@ function useCanvas() {
 				setCursorMode(fabricCanvas, 'default', CanvasType.draw, true);
 			}
 			canvas.current.discardActiveObject();
-			canvas.current.renderAll();
+			canvas.current.requestRenderAll();
```

Comment:
renderAll에서 requestRenderAll로 바꾸신 이유가 있으신가요?

Reply:
문서 찾아보니까 requestRenderAll가  next animation frame에 renderAll을 붙이는거고, renderAll는 강제로 렌더하는것으로 이해했는데 기본적으로 "performance boost when adding/removing a lot of objects to/from canvas at once since the renders are quequed and executed one per frame" 퍼포먼스 이점을 위해서는 requestRenderAll를 사용하는게 좋다고 생각했습니다.

http://fabricjs.com/docs/fabric.Canvas.html#renderOnAddRemove